deployment/media-insights-stack.yaml

```diff
@@ -148,8 +143,21 @@ Resources:
             Version: 2012-10-17
             Statement:
               - Effect: Allow
-                Action: ['logs:*']
-                Resource: 'arn:aws:logs:*:*:*'
+                Action:
+                  - logs:CreateLogGroup
+                  - logs:CreateLogStream
+                  - logs:PutLogEvents
+                Resource:
+                  - !Join [
+                    "",
+                    [
+                      "arn:aws:logs:",
+                      Ref: "AWS::Region",
+                      ":",
+                      Ref: "AWS::AccountId",
+                      ":log-group:/aws/lambda/*",
+                    ],
+                  ]
```

Comment:
if you don't mind leaving a comment, i think using `!Sub` is neater

Reply:
I agree, but this format is used elsewhere in the stack and I think it's best to maintain consistency.